test/Main.hs

```diff
@@ -56,7 +56,7 @@ main = do
   getTime <- mkAutoUpdate defaultUpdateSettings { updateAction = getCurrentTime }
 
   testDbConn <- getEnvVarWithDefault "POSTGREST_TEST_CONNECTION" "postgres://postgrest_test@localhost/postgrest_test"
-  setupDb testDbConn
+  resetDb testDbConn
```

Comment:
Resetting the data at the start is not needed right? The old `setupDb` didn't run `data.sql` as well. I've tested it and I think it's safe to remove.

```suggestion
```

Reply:
Yes, that would be even cleaner! I think this will require that nobody runs the tests on the same database twice - which is probably reasonable. Maybe we would need to document this for the users of `test/create_test_db`?

I would then also remove the `resetDB` and `loadFixture` functions in `SpecHelper`, no?

Reply:
Ah, no, those are still needed for the resets in line 93 in Main.hs. It should also be fine for reruns as far as I can see!

Reply:
Yes, some tests mutate the db and the `reset` function handles that. I think reruns will be fine.